addons/sms/data/ir_cron_data.xml

```diff
@@ -6,7 +6,7 @@
         <field name="state">code</field>
         <field name="code">model._process_queue()</field>
         <field name="user_id" ref="base.user_root"/>
-        <field name="interval_number">1</field>
+        <field name="interval_number">24</field>
```

Comment:
SMS scheduler seems to run on 10K records, and I don't see any resend / progress stuff. Not sure it is a good idea to have to wait 24 hours for the next batch if somehow you want to send more than 10K SMS (which can happen in sms marketing).

Reply:
I will create another PR to add progress to the SMS cron.